### DISCUSSION TITLE
Kube-bench v0.6.11 Released!

### DISCUSSION THREAD

**Discussion by chen-keinan** [aquasecurity/kube-bench#1372] [Announcements]
## Notable changes
- Support CIS Kubernetes V1.24 Benchmark v1.0.0 (#1329)
- Support customize datadir locations of etcd (#1330)

## Notable fixes
- Fix to empty grep and other cis-1.6-k3s checks (#1352)
-  False negative when audit_config is defined along with audit and config file not found (#1367)
- Use $etcddatadir in more etcd related checks (#1331

**Comment by GitHub13555997029** [comment]
What's the difference between Trivy's K8S scan and this one

**Reply by chen-keinan** [reply]
`trivy k8s` and `trivy operator` provide a complete scanning capability for k8s cluster

such as, vulnerabilities , misconfiguration, rbac assessment , exposed secret and infra assessment + compliance reports (NSA/CIS)
